src/sbmlutils/report/sbmlinfo.py

```diff
@@ -610,28 +587,29 @@ def info_events(self) -> List[Dict[str, Any]]:
             info = self.info_sbase(event)
 
             trigger = event.getTrigger()
-            info[
-                "trigger"
-            ] = f"""
-                {math(trigger, self.math_render)}
-                <br />initialValue = {trigger.initial_value}
-                <br /> persistent = {trigger.persistent}
-                """
-
-            priority = empty_html()
+            info["trigger"] = {
+                "math": math(trigger, self.math_render),
+                "initial_value": trigger.initial_value,
+                "persistent": trigger.persistent
+            }
+
+            priority = ""
             if event.isSetPriority():
                 priority = event.getPriority()
             info["priority"] = priority
 
-            delay = empty_html()
+            delay = ""
```

Comment:
This should not be empty strings, but None !.
Also use the following pattern consistently for attributes
```
info["delay"] = event.getDelay() if event.isSetDelay() else None

```